server/server.js

```diff
@@ -8,7 +8,6 @@ import next from "next";
 import Router from "koa-router";
 import session from "koa-session";
 import * as handlers from "./handlers/index";
-
```

Comment:
are the spacing changes here a result of the code transforms?

Reply:
Yeah it's from prettier. It should be this way, according to prettier.

Reply:
Weird, but okay 👌